go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c // indirect
 	gopkg.in/alexcesaro/statsd.v2 v2.0.0 // indirect
 	gopkg.in/fsnotify.v1 v1.4.7 // indirect
+	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
```

Comment:
Ahaha this has shows us something kinda funny. Why is square/go-jose still showing up in our indirect dependencies? Because the go-jose package has two subpackages (specifically `/cipher` and `/json`) which it imports... and the go-jose/go-jose version of the repo [hasn't re-written those import statements](https://github.com/go-jose/go-jose/blob/a10ff54e00bc6e833bf549e04ae976f0fe8ea2fd/crypter.go#L26)!

I think this is totally fine -- this PR is still a step in the right direction, and it'll make future version upgrades easier. Just amusing to note.

Reply:
I saw that too, but was not sure how you figured out `/cipher` and `/json` still come from `square/go-jose`. I thought it may have been some other package we use is using `square/go-jose`.

Reply:
We could fix those statements and then make a v2.6.1 tag.

Reply:
I figured it out by looking at modules.txt, which still lists
```
gopkg.in/square/go-jose.v2/cipher
gopkg.in/square/go-jose.v2/json
```

Updating the v2 branch to reference itself, tagging 2.6.1, and updating our dependency here is definitely a solution. Personally, I think using your other PR to skip straight to v3.x is the better ("more holistic") solution.

Reply:
The go-jose/go-jose v2 branch already completed this work via
- https://github.com/go-jose/go-jose/commit/a0568d778a639669bee1c4b5b42f363b76a7369d
- https://github.com/go-jose/go-jose/commit/28539eef3b7a9ae542fc631406272101f1896264

When do you think would be an appropriate time to start using v3 instead of using the proposed v2.6.1?